media-gfx/converseen/converseen-0.12.2.4.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Batch image converter and resizer based on ImageMagick"
+HOMEPAGE="https://converseen.fasterland.net/
+	https://github.com/Faster3ck/Converseen/"
+SRC_URI="https://github.com/Faster3ck/Converseen/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/${P^}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="graphicsmagick debug"
+
+RDEPEND="
+	dev-qt/qtbase:6[gui,network,widgets]
+	graphicsmagick? ( media-gfx/graphicsmagick:=[cxx,imagemagick] )
+	!graphicsmagick? ( media-gfx/imagemagick:=[cxx] )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-qt/qttools[linguist]
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-appdata-path.patch"
+	"${FILESDIR}/${PN}-0.9.9.0-no-update.patch"
+	"${FILESDIR}/${PN}-0.12.0.1-graphicsmagick-support.patch"
+)
+
+MYCMAKEARGS="-DUSE_QT6=yes"
```

Comment:
That's not how we do it - this overrides a user-defineable variable. Set it in `local mycmakeargs=( ... )` inside `src_configure()` phase.